masu/processor/orchestrator.py

```diff
@@ -35,16 +35,40 @@ class Orchestrator():
 
     """
 
-    def __init__(self):
+    def __init__(self, billing_source=None):
```

Comment:
This seems counter to our discussions that orchestrator is expected to enumerate across all accounts and all billing sources.

What's the intent here?